ecs_fargate/README.md

```diff
@@ -16,14 +16,13 @@ The ECS Fargate check is packaged with the Agent, [run the Agent][1] with your c
 
 #### Metric Collection
 
-1. Edit the `ecs_fargate.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your ECS Fargate performance data.
-    See the [sample ecs_fargate.d/conf.yaml][6] for all available configuration options.
+1. Edit the `ecs_fargate.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your ECS Fargate performance data. See the [sample ecs_fargate.d/conf.yaml][6] for all available configuration options.
```

Comment:
This would be quite cumbersome to do manually, as the container can't be updated once it runs, so users would need to maintain their own image. Instead we have a script that does that automatically plus a few more things if they set the `ECS_FARGATE` environment variable in the agent container. The script is [here](https://github.com/DataDog/datadog-agent/blob/6a0668670fe37784b09073556fd198027e477e2d/Dockerfiles/agent/entrypoint/50-ecs.sh), and an example of how to pass that env variable can be found in the [fargate blog post](https://www.datadoghq.com/blog/monitor-aws-fargate/#deploying-the-datadog-agent-on-aws-fargate)